lib/ansible/plugins/lookup/config.py

```diff
@@ -138,9 +138,6 @@ def run(self, terms, variables=None, **kwargs):
         if (ptype or pname) and not (ptype and pname):
             raise AnsibleOptionsError('Both plugin_type and plugin_name are required, cannot use one without the other')
 
-        if not isinstance(missing, string_types) or missing not in ['error', 'warn', 'skip']:
```

Comment:
This is already being handled by the config system.

Reply:
no, this is specific to lookup error handling options